presto-main/src/main/java/com/facebook/presto/server/ServerConfig.java

```diff
@@ -38,6 +40,30 @@ public ServerConfig setCoordinator(boolean coordinator)
         return this;
     }
 
+    public boolean isEnabledUIonSecondaryPort()
+    {
+        return enabledUIonSecondaryPort;
+    }
+
+    @Config("http-server.ui.secondary.port.enabled")
```

Comment:
I would like to do but I don't think it would be easy. I have to go through all the way to understand how to distinguish the property from the config file and the property from default value because I don't think set a wired value for default just to deduce how the property come from is a good one. Giving this property is just same as Facebook guys did. See here https://github.com/airlift/airlift/blob/master/http-server/src/main/java/io/airlift/http/server/HttpServerConfig.java#L94